### ISSUE TITLE
[Suggestion] Create a match/board simulator

### ISSUE THREAD

**Issue by rodrigo815** [magefree/mage#12505]
Sometimes we wish to rapidly check how some cards work together and it would be great to be able to create a board the way we want, with an imaginary opponent also controlled by us. Managing everything manually: everyone's board/hand/life point/counters/emblems/etc... But having **rule enforcement** with the press of a button. A play/pause button, maybe.

This would be a great feature for beginners, but also great for content creators and people who want to teach MTG.

**Comment by JayDi85** [comment]
You can do it with cheat mode - it allows to setup any board state in single or multiplayer games. See docs on wiki page: https://github.com/magefree/mage/wiki/Development-Testing-Tools

**Comment by rodrigo815** [comment]
Awesome! I'll try this right now.

**Comment by tiera3** [comment]
Is there any chance that cheat mode could be enabled during a normal game - requiring confirmation from the opponent in the same way that a rollback request requires confirmation. This would be useful in a game where a bug has been identified to correct the boardstate. (In most circumstances it wouldn't be needed, but in the rare case that something goes awry, this would be useful.)

**Comment by JayDi85** [comment]
Nope, really manual mode impossible due game engine. Every movement between zones raise events, triggers, etc. Same for any other actions like counters remove, life changing, etc. You can’t skip it.
